Mage/src/main/java/mage/abilities/AbilitiesImpl.java

```diff
@@ -102,7 +102,7 @@ public AbilitiesImpl(final AbilitiesImpl<T> abilities) {
                     rules.add(sbRule.toString());
                 }
                 String rule = ability.getRule();
-                if (rule.length() > 0) {
```

Comment:
Sure, you can add a server log message and set the text to something like "Error in rule text generation of " + this.class.getName() + ": Create a bug report of fix the source code"